src/main/java/com/czertainly/core/service/cmp/util/OIDS.java

```diff
@@ -0,0 +1,469 @@
+package com.czertainly.core.service.cmp.util;
+
+import java.util.Locale;
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+
+public enum OIDS {

```

Comment:
Why we have list of OIDs when it is available already in BouncyCastle? Do we need to maintain our own list of OIDs? It seems to me like duplicate and should be removed.

Reply:
Because of modified AlgorithmUtil, this class is obsolete (=removed)